### PR TITLE
Completed typescript-functions - Nader Motamedi

### PR DESCRIPTION
**Pull request by nmotamedi**
Screen Shot 2024-01-31 at 9 59 18 AM

### CODE REVIEW

File: typescript-functions/main.ts [hunk 1]
```diff
@@ -0,0 +1,68 @@
...
+console.log('convert 60 minutes to seconds: ', convertMinutesToSeconds(60));
+
+function greet(name: string): string {
+  const greeting: string = 'Hey ' + name + '!';
+  return greeting;
+}
+
+console.log('Greeting for Tim: ', greet('Tim'));
+
+const getArea: any = (width: number, height: number): number => {
```
Comment (shawnkost) [inline typescript-functions/main.ts:20]: Drop the `any` here. We only want to include types for the parameters and the return type. By adding `any` it's basically overriding your parameter and `return` types and not providing any type validation. Here is an example of our solution: 

```
const getArea = (width: number, height: number): number => {
  const area: number = width * height;
  return area;
};
```

File: typescript-functions/main.ts [hunk 2]
```diff
@@ -0,0 +1,68 @@
...
+  return width * height;
+};
+
+console.log('Area of width 4 and height 5: ', getArea(4, 5));
+
+const getFirstName: any = (person: Person): string => {
```
Comment (shawnkost) [inline typescript-functions/main.ts:26]: Same note above about `any`

**Review by shawnkost — APPROVED** [review]
Approved, but check out my comments about `any`

**Comment by nmotamedi** [comment]
Updated commit.

**Pull request merged**